CONTRIBUTING.md

```diff
@@ -28,6 +28,8 @@ We've tried to make editing an existing, public file as simple as possible.
 
 4. Using Markdown language, make your changes to the topic. For info about how to edit content using Markdown, see:
     - **If you're linked to the Microsoft organization in GitHub:** [Windows Open Publishing Guide Home](http://aka.ms/windows-op-guide)
+    
+    - **If you're external to Microsoft:** [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) 
```

Comment:
Looks like the list item got duplicated here

Reply:
Ah, it was a merge conflict with the HTTP -> HTTPS pr

Reply:
yes, I made a mistake in the conflict resolution when I fixed it. I did a PR to fix my mistake. Should be OK now. If not let me know.